javascript/javascript1/week1/Housey-pricey.js

```diff
@@ -0,0 +1,51 @@
+//Declaring all variables
+let width;
+let height;
+let depth;
+let volumeInMeters;
+let gardenSizeInM2;
+let paidPrice;
+let peterActualHouseCost;
+let juliaActualHouseCost;
+
+//peter House Price
+width = 8;
+height = 10;
+depth = 10;
+gardenSizeInM2 = 100;
+volumeInMeters = width*height*depth;
+paidPrice = volumeInMeters*2.5*1000 + gardenSizeInM2*300;
+console.log("peter has paid:"+" "+paidPrice);
+
+peterActualHouseCost = 2500000;
```

Comment:
Since this is only set once you should assign it directly as a const:
`const peterActualHouseCost = 2500000;`

Same goes for `juliaActualHouseCost`